doc/type_requirements/Hasher.qbk

```diff
@@ -7,37 +7,32 @@
 
 [section:Hasher Hasher]
 
-A [@Hasher] implements a hash algorithm. This is used to compute the small
-digests NuDB needs to effectively implement a hash table. NuDB provides
-the default implementation [link nudb.ref.nudb__xxhasher xxhasher], which is
-suitable for most use cases. For advanced applications, a user supplied
-hash function may be supplied which must meet these requirements.
+A type [@Hasher] implements a hash algorithm. This is used to compute the small
+digests NuDB needs to effectively implement a hash table.
 
-In the table below:
-
-* `X` denotes a hasher class
-* `a` denotes a value of type `X const`
-* `s` denotes a value of type `std::uint64_t`
+[heading Notation]
+* `H` is a hasher class
+* `h` is an instance of type `H`
+* `s` is a random seed of type `std::uint64_t`
 * `p` denotes a value of type `void const*`
 * `n` denotes a value of type `std::size_t`
 
-[table Hasher requirements
+[heading Valid Expressions]
+[table
 [[operation] [type] [semantics, pre/post-conditions]]
```

Comment:
Its kind of annoying that "Valid Expressions" appears twice in a row. Once for the heading and then again for the table name. See:
http://vinniefalco.github.io/stage/nudb/ramey/nudb/type_requirements/File.html